install_config/oab_broker_configuration.adoc

```diff
@@ -740,3 +740,111 @@ secrets:
   secret: db_creds
   apb_name: dh-rhscl-postgresql-apb
 ----
+
+[[configuring-oab-proxy]]
+== Running Behind a Proxy
+
+When running the OAB inside of a proxied {product-title} cluster, it is
+important to understand its core concepts and consider them within the context
+of a proxy used for external network access.
+
+As an overview, the broker itself runs as a pod within the cluster. It has a
+requirement for external network access depending on how its registries have
+been configured.
+
+[[configuring-oab-proxy-prerequisites]]
+=== Prerequisites
+
+Some important prerequisites necessary for running the broker behind a proxy:
+
+[[configuring-oab-proxy-cluster-proxy]]
+==== Configuring the Cluster to Run Behind a Proxy
+
+Before attempting to run the OAB behind a proxy, review
+xref:../install_config/http_proxies.adoc#install-config-http-proxies[Working
+with HTTP Proxies] and ensure your cluster is configured accordingly to run
+behind a proxy.
+
+In particular, the cluster must be configured to _not_ proxy internal cluster
+requests. This is typically configured with a `NO_PROXY` setting of
+`.cluster.local,.svc`, in addition to any other desired `NO_PROXY` settings.
+This is because the broker must be able to directly communicate with its etcd
+instance.
+
+[NOTE]
+====
+Brokers deploying unversioned, or v1 APBs _must_ also add `172.30.0.1` to their
+`NO_PROXY` list. APBs prior to v2 extracted their credentials from running APB
+pods via an `exec` HTTP request, rather than a secret exchange. Unless you are
+running a broker with experimental proxy support in a cluster prior to
+{product-title} 3.9, you probably do not have to worry about this.
+====
+
+[[configuring-oab-proxy-adapter-whitelists]]
+==== Registry Adapter Whitelists
+
+The broker's configured registry adapters must be able to communicate with their
+external registries in order to bootstrap successfully and load remote APB
+manifests. These requests can be made via the proxy, however, the proxy must
+ensure that the required remote hosts are accessible.
+
+Example required whitelisted hosts:
+
+[options="header"]
+|===
+|Registry Adapter Type |Whitelisted Hosts
+
+|`rhcc`
+|`registry.access.redhat.com`, `access.redhat.com`
+
+|`dockerhub`
```

Comment:
The original content had `dh` here, though in the OCP doc we've only mentioned the `dockerhub` type. Are either acceptable?

Reply:
There are two fields in a registry configuration, the `type`. In this case, only `dockerhub` is acceptable. The name defaults to `dh`, but this can be an arbitrary value. `dh`, or another relatively short name is preferred, as it's used as part of the `ServiceClass` label, which can become quite long.